Library/Formula/gdk-pixbuf.rb

```diff
@@ -52,12 +52,14 @@ def install
   def post_install
     # Change the version directory below with any future update
     ENV["GDK_PIXBUF_MODULEDIR"]="#{HOMEBREW_PREFIX}/lib/gdk-pixbuf-2.0/2.10.0/loaders"
+    ENV["GDK_PIXBUF_MODULE_FILE"]="#{lib}/gdk-pixbuf-2.0/2.10.0/loaders.cache"
```

Comment:
Interestingly enough, the bottle does contain a `loaders.cache`; so that symlink does get created.  By setting `GDK_PIXBUF_MODULE_FILE` as I have it up there, the original file gets destroyed and recreated by `gdk-pixbuf-query-loaders`, rather than the symlink getting destroyed and then recreated as a normal file.  I have updated two dependent formulae; `librsvg` passes its tests with this modification, I'm not sure about `xplanetfx`.